packages/deep/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@solid-primitives/deep",
```

Comment:
Wondering if the package shouldn't just be called `@solid-primitives/watch` / `@solid-primitives/on` / `@solid-primitives/track` and provide utilities for tracking / limiting sources.
Such as `defer` from utils or `on` from solid (there is an idea to move it to solid-primitives)

But it's fine for now - that's something to figure out in the future.